docs/topics/mta-view-questionnaire.adoc

```diff
@@ -5,13 +5,13 @@
 
 :_content-type: PROCEDURE
 [id="mta-view-questionnaire_{context}"]
-= Viewing a questionnaire
+= Viewing an assessment questionnaire
 
-.Procedure
+You can display the questions contained in the assessment questionnaire with the answer choices and weight displayed in the drop-down menu.
```

Comment:
Suggestion
Since there is no drop-down, the above statement could be rephrased as: 
You can view the questions in a questionnaire along with the answer choices and their associated risk weight.

Reply:
Will give it a thought, thanks :-)

Reply:
BTW, the last step in this procedure must be rewritten as well right? It mentions the drop-down menu. Can you please guide me on that?